src/main/java/XbeLoader/XbeXbSymbolDatabaseAnalyzer.java

```diff
@@ -120,14 +120,13 @@ public boolean added(Program program, AddressSetView set, TaskMonitor monitor, M
 			}
 			
 			exec.waitFor();
-		} catch (InterruptedException e) {
-			log.appendMsg("Failed to run " + toolExec);
-			return false;
-		} catch (IOException e) {
-			log.appendMsg("Failed to run " + toolExec);
-			return false;
-		} catch (InvalidInputException e) {
-			log.appendMsg("Failed to run " + toolExec);
+		} catch (Throwable e) {
+			int st_i = 0;
+			for (StackTraceElement stackTrace : e.getStackTrace()) {
+				log.appendMsg("stack[" + st_i + "]     : " + e.getStackTrace()[st_i].toString());
```

Comment:
Huh, I noticed a code usage error here. I should be using `stackTrace.toString()` instead of longer code `e.getStackTrace()[st_i].toString()`. I will fix this soon.